docker-compose.e2etests-local.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+services:
+  e2etests-local:
+    build:
+      context: .
+      dockerfile: src/e2etests/Dockerfile
+    depends_on:
+      - "ursadb"
+      - "dev-web"
+      - "dev-daemon"
+      - "redis"
+    volumes:
+      - "./src/utils/e2e-state/samples:/mnt/samples"
```

Comment:
`src/utils/e2e-state` - here and everywhere else - just `./e2e-state` is probably better